src/code/jpegdecoder.c

```diff
@@ -154,7 +154,7 @@ s32 JpegDecoder_ParseNextSymbol(JpegHuffmanTable* hTable, s16* outCoeff, s8* out
     if (sym) {
         *outCoeff = JpegDecoder_ReadBits(sym);
         if (*outCoeff < (1 << (sym - 1))) {
-            *outCoeff += (-1 << sym) + 1;
+            *outCoeff += (-1U << sym) + 1;
```

Comment:
So no `0xFFFFFFFF`?

Reply:
It's debatable but cf the commit message f4c7bf59777e5d08d41238908877d8033bb85c30

Reply:
I think that kind of information shouldn't get lots in a commit's message, would be nice to actually have it on the repo

Reply:
I could add a `// See ISO/IEC 10918-1, this part is "F.2.2.1 Huffman decoding of DC coefficients"` somewhere but not sure exactly
And like to be clear I'm only preferring "-1" here because I expect that's what the original code had (based on the standard having it), 0xFFFFFFFF would be fine but possibly misleading in recognizing the algorithm? 

The jpeg handling in general would benefit from a pass by someone who knows what they're doing / is willing to study the standard, across z_jpeg.c, jpegdecoder.c and the jpeg ucode, but that's probably not going to be me

Reply:
I think that comment sounds fine.
Alternatively you could do something like `// -1U is preferred over 0xFFFFFFFF because that's what the jpeg standard uses` or something